qemu/tests/cfg/virtio_net_dpdk.cfg

```diff
@@ -73,9 +73,10 @@
     env_hugepages_cmd = echo 8 >  /sys/devices/system/node/node0/hugepages/hugepages-1048576kB/nr_hugepages
     env_pkg = tuned-profiles-cpu-partitioning dpdk dpdk-devel dpdk-tools
     nic_driver = Virtio 82599
-    nic1_dpdk_driver = /usr/lib64/librte_pmd_virtio.so
-    nic2_dpdk_driver = /usr/lib64/librte_pmd_ixgbe.so
+    nic1_dpdk_driver = /usr/lib64/dpdk-pmds/librte_net_virtio.so.21.0
+    nic2_dpdk_driver = /usr/lib64/dpdk-pmds/librte_net_ixgbe.so.21.0
```

Comment:
I will check if we need to keep RHEL7 support and comment later here.

Reply:
Seems you could decide it for yourself per your test requirement, it you don't need to test RHEL7 guest anymore, then it would be fine.

Reply:
we do not need test RHEL7 guest for this. thanks.